spec/requests/admin_mentors_controller_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+RSpec.describe Admin::MentorsController, type: :request do
+
+  let(:auth_headers) do
+    { "Authorization" => "Bearer #{@token}" }
+  end
+
+  let!(:record) { create(:user, :mentor)}
+
+  context "as admin" do
+    before do
+      user = create(:user, :admin)
+      @token = Knock::AuthToken.new(payload: user.to_token_payload).token
+    end
+
+    it "GET index" do
+      get "/admin/mentors", headers: auth_headers
+      expect(response.status).to eq 200
+    end
+
+    it "GET show" do
+      get "/admin/mentors/#{record.id}", headers: auth_headers
+      expect(response.status).to eq 200
+    end
+
+    it "POST create" do
+      attributes = attributes_for(:user, :mentor)
+
+      post "/admin/mentors", params: { user: attributes }, headers: auth_headers
+      expect(response.status).to eq 201
+
+      attributes.except(:password).each do |attr, val|
```

Comment:
Check if we can use `should eq` with `with_indifferent_access` and `except("id")`

Reply:
We can, but I feel this is a worse path, as we'd have to do something like

```
expect(attributes.except(:password)).to eq json.except(:id, :bio, :picture, :admin, :active, :mentor, :blocked, :career_ids, :trait_ids)
```

So I'm leaving it as is for now